### ISSUE TITLE
What is a selection matrix?

### ISSUE THREAD

**Issue by Maxlyu254** [dilevin/computer-graphics-mass-spring-systems#19]
In the header file _fast_mass_springs_precomputation_dense.h_, one of the specified outputs is C, which is described as "#b by #V selection matrix". What exactly is a selection matrix? I guess it is a matrix such that:

C(i, j) = 1, if the i-th entry of b (list of "pinned"/fixed vertices as indices into rows of V) is j
C(i, j) = 0, otherwise

Is this correct?

**Comment by Zhecheng-Wang** [comment]
Yes. Basically selection matrix is a matrix that "selects" rows of matrix $A$ upon multiplication with $A$.
